handbook/engineering/code-intelligence/index.md

```diff
@@ -14,9 +8,62 @@ The Code Intelligence team builds tools and services that provide contextual inf
 
 - Robust, extensible, and scalable infrastructure to index code across all languages, keep those indexes up-to-date, and efficiently resolve code intelligence queries against all indexed code.
 
+## Vision
+
+To be the point of discovery for the knowledge graph for all source code.
+
+## Mission
+
+Build a code intelligence platform that understands code in all languages and provides rich metadata to power code search, insights and campaigns. 
+
 ## Goals
 
-TODO
+**Provide precise code intel for the primary programming languages that combined cover > 50% of usage across GitHub.com repositories or across customer projects (greater of the two).**
```

Comment:
@nicksnyder I can speak to why its an 'or' statement: we have [incomplete information](https://docs.google.com/spreadsheets/d/1i3Edpz3NPqN1weTnr4pqEbKxxyNhgd96vMAq-R-5LII/view) for all of our customers in terms of language usage - if the set of customer data is too small to show significant usage (i.e. for 50 customers we only know information for 10 and when analyzing those 10 we can't sufficiently summize that usage represents > 50%) then fallback to a stat we _can_ get complete info for: github repositories.

We are not excluding GitLab intentionally - more that Github has already made their usage stats public see: [GitHub Octoverse](https://octoverse.github.com/) and [summary of language research](https://docs.google.com/spreadsheets/u/1/d/1o_OsbD6lgYS6XmMwuwXMSYddD5p2dKYqp0ZfklEs9hM/edit#gid=230720276), and  I wasn't able to find that info for GitLab.

Reply:
> usage stats based on GitHub report

From the Github report on language usage in 2019:
"unique contributions to public and private repositories in a language."

Reply:
Can you clarify why this is an or statement rather than summing across GitHub.com and customer projects?

If we do that GitHub usage will have significantly more influence then customer projects simply due to the fact that our customer data is spotty. The preference is to prioritize solving for existing customers first, then general usage.